.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
+indent_style = tab
+indent_size = 2
```

Comment:
Not sure why we should specify this, the whole point of using tabs is that everyone can use their own tab width. Tab width is a local setting, that does not affect the underlying code at all.

Reply:
Happy to remove it from in a bit.

I think a common reason to include it is for when max line length is set, it keeps things consistent. For example, if the repo mandates a max line length of 120, then the "length" of lines can vary depending on the indent size of developers code editors.

Ofc that doesn't apply here, honestly… I just left by mistake when copy and pasting it over from another project.

Reply:
Even if a line length was set, I think the flexibility of everyone having their preferred tab width outweighs the importance of having shorter line lengths (I could even argue that indentation should not count in line length, though having a small cyclomatic complexity is desirable, and that correlates with indentation).